modules/pcd/src/lib/parse-pcd.ts

```diff
@@ -10,15 +10,55 @@
 
 import {Schema, Field, Float32, Uint8, FixedSizeList} from '@loaders.gl/schema';
 import {getMeshBoundingBox} from '@loaders.gl/loader-utils';
-
-const LITTLE_ENDIAN = true;
-
-export default function parsePCD(data) {
+import {SchemaMetadata} from '@loaders.gl/schema/lib/schema/classes/schema';
```

Comment:
It's not OK to import from subpaths in the src code (it is only OK to do this in the tests).

Reply:
> It's not OK to import from subpaths in the src code (it is only OK to do this in the tests).

ok, switched to local type